pkg/model/planner.go

```diff
@@ -169,47 +169,65 @@ type workflowPlanner struct {
 }
 
 // PlanEvent builds a new list of runs to execute in parallel for an event name
-func (wp *workflowPlanner) PlanEvent(eventName string) *Plan {
+func (wp *workflowPlanner) PlanEvent(eventName string) (*Plan, error) {
 	plan := new(Plan)
 	if len(wp.workflows) == 0 {
-		log.Debugf("no events found for workflow: %s", eventName)
+		log.Debugf("no workflows found for workflow: %s", eventName)
```

Comment:
This should probably be

```suggestion
		log.Debugf("no workflows found for event: %s", eventName)
```